.github/workflows/pull_request.yaml

```diff
@@ -30,24 +36,26 @@ jobs:
           eslint_extensions: ts
 
       - name: OpenAPI Lint Checks
+        if: ${{ github.event.inputs.openApi.required == 'true' }}
```

Comment:
This condition is wrong. You should add another argument (other than `openApiFilePath`) and execute this step only if this variable's value is `true`.

Reply:
Did you mean to have an argument of the openApiFilePath and an argument of the openApi file name?